src/containers/App.tsx

```diff
@@ -194,6 +194,7 @@ const Content = (): JSX.Element => {
     const user = useFirebaseAuthentication()
     const settings = useSettings()
     const location = useLocation()
+    const [openTavle] = useState(new Date().getTime())
```

Comment:
Could be rename to something more descriptive. Maybe `tavleOpenTime` or `tavleOpenedAt`.
```suggestion
    const [tavleOpenTime] = useState(new Date().getTime())
```

Reply:
Renamed to `tavleOpenedAt`